Formula/mimic.rb

```diff
@@ -0,0 +1,31 @@
+class Mimic < Formula
+  desc "Mycroft's TTS engine, based on CMU's Flite (Festival Lite)"
+  homepage "https://mimic.mycroft.ai"
+  url "https://github.com/MycroftAI/mimic/archive/1.2.0.2.tar.gz"
+  sha256 "6adcc9911b09d6e9513add41ad9dfc0893ece277f556419869520a0f0708c102"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "icu4c"
+  depends_on "portaudio"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-shared",
+                          "--enable-static",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "check"
```

Comment:
We don't run testsuites or checks except for security software. Please remove this line.